lib/res/signup.js

```diff
@@ -26,6 +27,9 @@ function signup(req, res, next) {
   console.log(params);
```

Comment:
looks like this was here already, but do we want to leave it? is it necessary i mean? typically logs like this should be removed before merging